packages/buckets/src/buckets.ts

```diff
@@ -90,6 +97,47 @@ export class Buckets {
     return new Buckets(client.context)
   }
 
+  /**
+   * Open a new / existing bucket by bucket name and ThreadID
+   * @param name name of bucket
+   * @param threadID id of thread where bucket is stored
+   */
+  async open(name: string, threadName?: string, threadID?: ThreadID) {
+    if (threadID) {
```

Comment:
allows for directly picking threadid. but i'm going to move examples to just using default 'buckets' most of the time. so just `buckets.open(bucket_name)`